cob_cartesian_controller/include/cob_cartesian_controller/cartesian_controller_data_types.h

```diff
@@ -52,8 +51,9 @@ namespace cob_cartesian_controller
 
     struct MoveCircStruct
     {
-        double x_center, y_center, z_center, roll_center, pitch_center, yaw_center;
-        double start_angle, end_angle, radius;
+        geometry_msgs::Pose pose_center;
+        double start_angle, end_angle;
```

Comment:
No that's not necessary because it is like what I've expected and what is described in https://de.wikipedia.org/wiki/Internationales_Einheitensystem. Just wanted to get a clear statement.